code/style.css

```diff
@@ -1,4 +1,309 @@
+  position: relative;
+  flex-direction: column;
+  padding: 10px;
+  box-sizing: border-box;
+}
+
+
+h1 {
+  font-size: 15px;
+  margin: 0px 0px 10px 0px;
+  font-family: Arial, Helvetica, sans-serif
+}
+
+ .container-1 h2 {
+  padding: 30px 0px;
+  font-family: Quicksand;
+  font-size: 20px;
+} 
+
+
+.container-2 h2 {
+  padding: 20px 0px;
+  font-family: Quicksand;
+  font-size: 20px;
+} 
+
+
+.container-1 container-2 p{
+  font-size: 15px;
+  line-height: 32px;
+  font-family: Raleway;
+}
+
+.big-news img{
+  align-items: center;
+  width: 100%;
+  height: auto;
+  margin-inline-start: 20px;
+}
+
+.big-news h1{
+  background-color: black;
+  position: absolute;
+  color: white;
+  bottom: 75%;
+  width: 50%;
+  padding: 20px;
+}
+
+.big-news-text{
+  width: 100%;
+  margin-inline-start: 10px;
+  padding: 40px;
+}
+
+.container-2{
+  display: flex;
+  flex-direction: column;
+}
+
+.container-2 img{
+  width: 100%;
+  height: auto; 
+  margin-inline-start: 20px;
+}
+
+ .second-news-text{
+  align-items: flex-start;
+  width: 100%;
+  padding: 40px; 
+}
+
+.container-3{
+display: grid;
+grid-template-columns: repeat(1, 1fr);
+grid-template-rows: repeat(4, 120vw);
+grid-gap: 15px;
+} 
+
+.container-3 img{
+  width: 70%;
+  height: auto;
+}
+
+.blog-post {
+  font-family: Raleway;
+  margin: 5px;
+  text-align: center;
+}
+
+.blog-post h2{
+  font-size: 20px;
+  font-family: quicksand;
+}
+
+.blog-post p{
+  font-size: 15px;
+}
+
+.blog-post:hover {
+  width: 100%;
+  /* box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19); */
+  opacity: 0.8;
+}
+
+
+/*Tablet*/ 
+@media (min-width: 667px) and (max-width: 1024px){
+  body{
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .big-news h1{
+    font-size: 30px;
+    bottom: 45%;
+    width: 60%;
+  }
+
+  .container-2{
+    flex-direction: column;
+    padding: 10px;
+    margin: 30px;
+  }
+
+  .container-3{
+    display: grid;
+    grid-template-columns: repeat(2, 2fr);
+    grid-template-rows: repeat(4, 60vw);
+    } 
+}
+
+/* Computer layout*/
+@media (min-width: 1025px) {
+  body {
+    justify-content: left;
+}
+
+p{
+  font-size: 18px;
+  line-height: 32px;
+}
+
+h1 {
+  font-size: 30px;
+}
+
+.container-1 h2{
+  font-size: 30px;
+}
+
+.container-2 h2{
+  font-size: 30px;
+}
+
+.menu-button {
+  display: none;
+}
+
+.dropdown-content {
+  display: block;
+  position: static;
+  background: transparent;
+  box-shadow: none;
+}
+
+.dropdown-list {
+  display: flex;
+}
+
+.big-news h1{
+  background-color: black;
+  position: absolute;
+  color: white;
+  bottom: 40%;
+  width: 40%;
+  padding: 30px;
+}
+
+.container-1{
+  flex-direction: column;
+}
+
+.container-2{
+  flex-direction: row;
+  padding: 10px;
+  margin: 30px;
+}
+
+.second-news {
+  width: 80%;
+}
+
+.second-news-text{
+  padding: 20px 40px 40px 50px;
+  width: 800px;
+}
+
+ .container-3{
+  display: grid;
+  grid-template-columns: repeat(4, 2fr);
+  grid-template-rows: repeat(4, 30vw);
```

Comment:
Nice there with the grid!! If you change the repeat 4 to 2, you don't get all that empty space at the bottom on desktop.